xchange-bitstamp/src/main/java/org/knowm/xchange/bitstamp/dto/account/BitstampRippleDepositAddress.java

```diff
@@ -1,20 +1,28 @@
 package org.knowm.xchange.bitstamp.dto.account;
 
-import com.fasterxml.jackson.annotation.JsonIgnore;
 import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.core.ObjectCodec;
+import com.fasterxml.jackson.databind.DeserializationContext;
+import com.fasterxml.jackson.databind.JsonDeserializer;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import java.io.IOException;
 
-public class BitstampRippleDepositAddress {
+@JsonDeserialize(using = BitstampRippleDepositAddress.BitstampDepositAddressDeserializer.class)
+public class BitstampRippleDepositAddress extends BitstampDepositAddress {
 
   @JsonProperty("address")
   private final String addressAndDt;
 
-  @JsonIgnore private String address = null;
+  private String address = null;
 
-  @JsonIgnore private Long destinationTag = null;
+  private Long destinationTag = null;
 
-  public BitstampRippleDepositAddress(@JsonProperty("address") String addressAndDt) {
-
-    this.addressAndDt = addressAndDt;
+  public BitstampRippleDepositAddress(String error, String depositAddress) {
```

Comment:
this is done as extension of org.knowm.xchange.bitstamp.dto.account.BitstampDepositAddress, which is using this such DTO. I didn't want to change existing code, since this could break some production environments